application/templates/static_site/measure.html

```diff
@@ -12,7 +12,7 @@
 
 {% block pageTitle %}{{ measure_version.title }} - GOV.UK Ethnicity facts and figures{% endblock %}
 {% block socialTitle %}{{ measure_version.title }}{% endblock %}
-{% block socialDescription %}{{ measure_version.summary | first_bullet | render_markdown | striptags }}{% endblock %}
+{% block socialDescription %}{% if measure_version.description %}{{ measure_version.description }}{% else %}{{ measure_version.summary | first_bullet | render_markdown | striptags }}{% endif %}{% endblock %}
```

Comment:
@samuelhwilliams do you think it'd be worth abstracting this to the model?

Something like:

```python
    @property
    def social_description(self):

        if self.description:
            return self.description
        else:
            return first_bullet(self.summary)
```

Reply:
Seems reasonable. 👍

Reply:
Hmm, I guess the question then is which of `first_bullet`, `render_markdown` and `striptags` should be part of the model method? My hunch is just the first one, and then we could apply `render_markdown` and `striptags` in the template (even though that's semi-redundant for the manually written one).